tests/unit/RequestBuilder/CampaignRequestBuilderTest.php

```diff
@@ -23,27 +26,28 @@ public function shouldBuildRequestWithCommands(): void
     {
         $builder = new CampaignRequestBuilder();
 
-        $recommendationCommand1 = UserRecommendation::create('userId1', 'scenario1')
+        $recommendationCommand1 = UserItemRecommendation::create('userId1', 'scenario1')
             ->setCount(1)
             ->setRotationRate(1.0)
             ->setRotationTime(600);
 
-        $recommendationCommand2 = UserRecommendation::create('userId2', 'scenario2')
+        $recommendationCommand2 = UserUserRecommendation::create('userId2', 'scenario2')
             ->setCount(2)
-            ->setRotationRate(0.5)
-            ->setRotationTime(700);
+            ->setRotationRate(1.0)
+            ->setRotationTime(600);
+
+        $recommendationCommand3 = ItemUserRecommendation::create('itemId1', 'scenario3')
+            ->setCount(3);
 
-        $recommendationCommand3 = UserRecommendation::create('userId3', 'scenario3')
-            ->setCount(3)
-            ->setRotationRate(0.0)
-            ->setRotationTime(800);
+        $recommendationCommand4 = ItemItemRecommendation::create('itemId2', 'scenario3')
```

Comment:
```suggestion
        $recommendationCommand4 = ItemItemRecommendation::create('itemId2', 'scenario4')
```
(only for consistency)

Reply:
fixed